src/main/plugins/operating-system-commands-plugin/windows-operating-system-command-repository.ts

```diff
@@ -44,7 +44,7 @@ export class WindowsOperatingSystemCommandRepository implements OperatingSystemC
                     type: IconType.SVG,
                 },
                 name: this.translationSet.windowsRestart,
-                searchable: [this.translationSet.windowsRestart],
+                searchable: [this.translationSet.windowsRestart, "reboot"],
```

Comment:
What if I have the language set to something else than English? ;)

Reply:
For most languages, reboot and restart have the same word. For some that have different words, would it be a good idea to create a file with an array and import and use it here. 

OR 

Since this is specific issue related to only restart/reboot and not any other system commands, would you like me to create that array in this file itself?

Reply:
I think I get what you mean. Eg. If I have it set to czech, then reboot should not suggest restart since there isn't a word like "reboot" in czech.

I think the only solution to that is to add a windowsReboot to the `translation-set.ts` and change only the searchable to:
```suggestion
                searchable: [this.translationSet.windowsRestart, this.translationSet.windowsReboot],
```